pkg/storage/couchdb/couchdbstore_test.go

```diff
@@ -197,7 +203,8 @@ func TestProvider_OpenStore(t *testing.T) {
 
 		newStore, err := provider.OpenStore("")
 		require.Nil(t, newStore)
-		require.Equal(t, "kivik: dbName required", err.Error())
+		require.Error(t, err)
+		require.Contains(t, err.Error(), "kivik: dbName required")
```

Comment:
same here.. if `"kivik: dbName required"` is the full error string, just use `require.EqualError(t, err, "kivik: dbName required")`

and the same goes for most error string comparison in this patch..

Reply:
In this case, `kivik: dbName required` is not the full error string

Reply:
Actually, in a lot of cases now the full error strings are quite long, which is why you see me using that require.Contains() call... yes, sometimes that "contains" text is still pretty long.

Reply:
if you have a defined error variable containing the exact string: `"kivik: dbName required"`, then you can at least use `require.True(t, errors.Is(err, yourKiviErrVariableHere))`

Reply:
In this case, I don't have an error variable for `kivik: dbName required` - that's returned from the Kivik third-party library.

For some of the other raw strings, I do actually have variables defined in the errors.go file with the text, but it would be a relatively large amount of effort to use them in the unit tests... since they have %w operands in them. And there's so many...

I see a few options here... what do you think I should do?
1. I could use all the error message vars I've defined and "construct" the full errors in the unit tests to avoid the string duplication. This will be the "cleanest", but will involve a lot of time...
2. I could use require.Contains() to just check for the lowest level error. Quick, although less precise in terms of error testing.
3. Leave as is.

What do you think?

Reply:
if you don't have control over the error returned, then it's fine.. no need to add more changes.. it's just a test after all.